java/images/jboss/s2i/assemble

```diff
@@ -165,6 +165,7 @@ function build_maven() {
 
 echo "=================================================================="
 echo "Starting S2I Java Build ....."
+mkdir -p ${DEPLOYMENTS_DIR}
```

Comment:
Could you please protect this like in 

```
[ -d "${DEPLOYMENTS_DIR}" ] || mkdir -p "${DEPLOYMENTS_DIR}"
```

The reason is set if we should ever switch to set -e for a more strict error handling, then this would be already safe.

Also dont forget the quotes in case the directory name contains spaces.

Reply:
Of course, this comment should go to the original template source. you got the idea ;-)

Reply:
Done!  (PS: Not sure if I'm making a mess out of GitHub becaues I'm more used to the Gerrit workflow or if it's just dumb; don't know where the original commit went here and why it's not showing up on the Commit list; but it's definitely there on the branch that this PR is raised from.)